### ISSUE TITLE
Backend, back-end, or backend? Frontend, front-end or frontend?

### ISSUE THREAD

**Issue by stoobie** [redhat-documentation/supplementary-style-guide#182]
I don't know what IBM SG says, but...
- guidance from Google style guide is _backend_/_frontend_
- guidance from MS and Apple is "front end (n.), front-end (adj.) Note hyphenation of adjective."

The RH SG uses both (see [Release Notes](https://redhat-documentation.github.io/supplementary-style-guide/#release-notes)).

**Comment by mportman12** [comment]
From the ISG:
![Screenshot from 2022-05-11 08-09-29](https://user-images.githubusercontent.com/74915975/167846505-197f974b-204f-4e1d-8808-1321ff983eed.png)
![Screenshot from 2022-05-11 08-09-10](https://user-images.githubusercontent.com/74915975/167846514-1c26b352-8a99-4f58-baad-a735b94bf819.png)

TL;DR: ISG recommends using `back end`/`front end` and `back-end`/`front-end` as a noun and adjective respectively, except when `backend`/`frontend` are already part of established product terminology.

**Comment by stoobie** [comment]
Thanks, @mportman12 !
